services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -1414,4 +1414,4 @@ async def get_project_inactivity(
     project_inactivity = await director_v2_api.get_project_inactivity(
         app, project_id, max_inactivity_seconds
     )
-    return parse_obj_as(Envelope[GetProjectInactivityResponse], project_inactivity)
+    return Envelope(data=parse_obj_as(GetProjectInactivityResponse, project_inactivity))
```

Comment:
I do not get how this can work. Is this called by _crud_handlers.py? How can this even work? Aren't you doing a double envelope here?

Reply:
No it's not a double envelope. That I have tested and I am sure of. Also director-v2 does not create and envelope for this reply.

Reply:
Good point. And yes. I think it was double enveloped. How was this not caught in the tests

Reply:
I managed to trick myself twice.